build.gradle

```diff
@@ -181,12 +193,14 @@ tasks.register('systemtest', Test) {
 }
 
 def serveTestReportInBackground = tasks.register('serveTestReportInBackground', com.github.psxpaul.task.JavaExecFork) {
-    dependsOn zipReport, compileJava, processResources, classes
+    description 'Creates a background server process for the test report that is to be used by Cypress'
+    dependsOn zipReport, compileJava, processResources, copyCypressConfig, copyMainClasses
     tasks.compileJava.mustRunAfter(zipReport)
     tasks.processResources.mustRunAfter(zipReport)
```

Comment:
Can I check if we need to ensure that the `copyCypressConfig` and `copyMainClasses` tasks need to run in a particular order?

Reply:
Between these 2 tasks, there is no order in which they must be run, since the cypress config is only used at runtime.

Reply:
Sorry, I was referring to these tasks in relation to the other tasks when `serveTestReportInBackground` is executed.

Reply:
For `copyCypressConfig`, no particular order needed.

For `copyMainClasses`, it already depends on `classes`, which in turn depends on `compileJava` and `processResources`.

Reply:
I see. This is ok for now but I think we should document the Gradle tasks in the developer guide. It is getting more complicated and the task dependencies are not immediately obvious when looking at the code.